v63002/u_inref/gtm8165.csh

```diff
@@ -0,0 +1,27 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Testing YottaDB can support fractional timeout values to the millisecond for
```

Comment:
Seems like a copy-paste issue from somewhere unrelated.

Reply:
I realize this copy-paste is from v63002/gtm5250 subtest. There too we need to randomly choose a no-timeout scenario (should have same effect as the timeout scenario) instead of the timed scenario currently implemented.

Reply:
This comment about v63002/gtm5250 subtest does not seem to be addressed still.